lib/fuse_lowlevel.c

```diff
@@ -2333,9 +2333,21 @@ int fuse_lowlevel_notify_expire_entry(struct fuse_session *se, fuse_ino_t parent
 }
 
 int fuse_lowlevel_notify_inval_entry(struct fuse_session *se, fuse_ino_t parent,
-				     const char *name, size_t namelen)
+						 const char *name, size_t namelen)
+{
+	return fuse_lowlevel_notify_entry(se, parent, name, namelen, FUSE_LL_INVALIDATE);
```

Comment:
As far as I can tell, fuse_lowlevel_notify_entry silently ignores this flag value.

Reply:
this one is the "old" `fuse_lowlevel_notfiy_expire_entry` that just fallsback to `invalidate`.

My idea was to have  this as a (more performant) base function, only checking if `invalidate` is supported - and all other flags (existing or not) do an automatic fallback to `invalidate` if not supported.  If users want the proper check they should use the specific function named after the command they want to execute (e.g. `fuse_lowlevel_notfiy_expire_entry` to `expire` the entry)